src/components/DatePickerInline/YearPicker.tsx

```diff
@@ -19,7 +19,7 @@ export default function YearPicker({
     startYear: number;
     endYear: number;
 }) {
-    const { FlatList, View } = useMolecules();
+    const { FlatList, View, HorizontalDivider } = useMolecules();
     const yearPickerStyles = useComponentStyles('DatePicker_YearPicker');
     const flatList = useRef<FlatListRef<number> | null>(null);
     const years = range(isNaN(startYear) ? 1800 : startYear, isNaN(endYear) ? 2200 : endYear);
```

Comment:
abstract this and use first and last if you need it to disable. the left and right navigation arrows.